suricata.yaml.in

```diff
@@ -979,7 +979,7 @@ app-layer:
            # Specifies the maximum amount of decompressed data to obtain,
            # set 0 for unlimited.
            swf-decompression:
-             enabled: yes
+             enabled: no
```

Comment:
I think this needs some entry in doc//userguide/upgrade.rst

Reply:
Does this need a redmine ticket entry @jasonish ?

Reply:
Might be a good idea.  I think the reasoning for moving from enabled by default to disabled by default needs to be documented. So probably a ticket, and a small note in upgrade.rst.

Reply:
link to the ticket in the upgrade.rst? :)

Reply:
No problem. I will create a ticket for this an link it in the upgrade.rst.

Reply:
I ended up submitting a separate PR for this part.  #8153